gravitee-apim-rest-api/docker/Dockerfile-dev

```
@@ -20,12 +20,11 @@ LABEL maintainer="contact@graviteesource.com"
 ARG GRAVITEEIO_VERSION=0
 ENV GRAVITEEIO_HOME /opt/graviteeio-management-api
 
-ADD ./gravitee-apim-rest-api-distribution-${GRAVITEEIO_VERSION}.zip .
+ADD ./gravitee-apim-rest-api-distribution-${GRAVITEEIO_VERSION}.tar.gz /tmp/
```

I think we can directly put it at the right place:
```suggestion
ADD ./gravitee-apim-rest-api-distribution-${GRAVITEEIO_VERSION}.tar.gz ${GRAVITEEIO_HOME}
```

And remove:
```
     && mv /tmp/gravitee-apim-rest-api-distribution-${GRAVITEEIO_VERSION} ${GRAVITEEIO_HOME} \
```